Tasks/XamariniOS/task.json

```diff
@@ -47,6 +47,30 @@
             "helpMarkDown": "Standard configurations are Ad-Hoc, AppStore, Debug, Release."
         },
         {
+            "name": "clean",
+            "type": "boolean",
+            "label": "Clean",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Run a clean build (/t:clean) prior to the build."
```

Comment:
Hosted builds shouldn't notice the effect of this being `true` or `false`, but isn't the normal preference for this to be `true` for private agents?  Should we just default it to `true`?

Related thought:
Can Xamarin prior to Cycle 9 gracefully ignore `/t:clean` (since it is not supported) if this input enables it by default?